packages/workflow/src/NodeParameters/FilterParameter.ts

```diff
@@ -183,9 +196,9 @@ export function executeFilterCondition(
 				case 'notEndsWith':
 					return !left.endsWith(right);
 				case 'regex':
-					return new RegExp(right).test(left);
+					return new RegExp(parseRegexPattern(right)).test(left);
 				case 'notRegex':
-					return !new RegExp(right).test(left);
+					return !new RegExp(parseRegexPattern(right)).test(left);
```

Comment:
Do we need to wrap it in a second RegExp since parseRegexPattern already returns RegExp?

Reply:
Cheers, Have fixed that